Example Apps/Example Swift/MenuManager.swift

```diff
@@ -206,6 +207,46 @@ private extension MenuManager {
             })
         })
     }
+
+    /// Menu item that shows remote control example
+    ///
+    /// - Parameter manager: The SDL Manager
+    /// - Returns: A SDLMenuCell object
+    class func menuCellRemoteControl(with manager: SDLManager, remoteManager: RemoteControlManager) -> SDLMenuCell {
+        var submenuItems = [SDLMenuCell]()
+        let errorMessage = "Changing the template failed"
+
+        /// Climate Control Menu
+        submenuItems.append(SDLMenuCell(title: ACRemoteControlClimateMenuName, secondaryText: nil, tertiaryText: nil, icon: nil, secondaryArtwork: nil, voiceCommands: nil, handler: { (triggerSource) in
```

Comment:
Use this image for this item (adding it to the assets should make it available in both apps?)

![iconmonstr-control-panel-21-240](https://user-images.githubusercontent.com/1767063/182884569-ac8271c9-b102-4b72-9cc1-8166370c670b.png)